### PR TITLE
Feature/pool model endpoints

### PR DESCRIPTION
**Pull request by TheRealSeber**

### CODE REVIEW

**Comment by TheRealSeber** [comment]
idk how to get a data from the relationship using the sqlalchemy

and i am mad af.

**Comment by matined** [comment]
> idk how to get a data from the relationship using the sqlalchemy
> 
> and i am mad af.

Let's return only model_id, we'll add name as a separate task as it's harder and we'll have to do it in a few places

File: app/api/tests/test_config/__init__.py
Comment (matined) [inline app/api/tests/test_config/__init__.py:1]: dont we already have `/tests/config` and `tests/db` folders? Is it a mistake?

**Review by matined — CHANGES_REQUESTED** [review]

**Review by matined — APPROVED** [review]

**Comment by TheRealSeber** [comment]
Nabijanie commitow na profilu >>>>

**Pull request merged**